upb_generator/protoc-gen-upbdefs.cc

```diff
@@ -77,7 +82,8 @@ void WriteDefHeader(upb::FileDefPtr file, Output& output) {
       "#endif\n\n",
       ToPreproc(file.name()));
 
-  output("extern _upb_DefPool_Init $0;\n", DefInitSymbol(file));
+  output("extern $1_upb_DefPool_Init $0;\n", DefInitSymbol(file),
```

Comment:
Nit: While it is a nice property that the codegen doesn't have 2 spaces when the param isn't passed, having the $1 attached to the symbol names looks too much to me like its actually trying to concatenate there.

I think if we want to avoid double-spaces it might be clearer if you attach the $1 to the extern (and PadPrefix instead of PadSuffix), since it'll be more clear that its not actually concatenating to make a type name. What do you think?

Reply:
Yes, I think that is less confusing - will change.